sky/utils/controller_utils.py

```diff
@@ -1352,7 +1352,7 @@ def _get_parallelism(pool: bool, raw_resource_per_unit: float) -> int:
     resource_per_unit = ratio * (raw_resource_per_unit +
                                  resource_per_unit_worker)
 
-    return max(int(total_memory_mb / resource_per_unit), 1)
+    return max(int(total_memory_mb / resource_per_unit), 0)
```

Comment:
Seems this breaks some managed jobs tests. I think we need to avoid this until we can fail early, because the current failure mode (infinitely pending) is pretty bad.